src/menu/menu.styles.tsx

```diff
@@ -0,0 +1,28 @@
+import styled from "styled-components";
+import { Border, Colour, Radius } from "../theme";
+
+// =============================================================================
+// STYLES INTERFACE
+// =============================================================================
+interface MenuPanelStyleProps {
+    $maxHeight?: number | undefined;
+}
+
+// =============================================================================
+// MENU STYLES
+// =============================================================================
+export const MenuPanel = styled.div<MenuPanelStyleProps>`
+    border-radius: ${Radius["md"]};
+    border: ${Border["width-010"]} ${Border["solid"]} ${Colour["border"]};
+    background: ${Colour["bg"]};
+    box-shadow: 0 ${Border["width-020"]} ${Border["width-080"]} 0
```

Comment:
box shadow can be hardcoded for now, there will be separate tokens for those in the future